### ISSUE TITLE
[develop branch] Click and dragging node in patchbay causes segfault

### ISSUE THREAD

**Issue by pdesaulniers** [falkTX/Carla#801]
If I run carla with the data/valgrind.sh script and click-and-drag an events-out node, I get a crash + this output (not sure if relevant):
```
CarlaPipeCommon::_writeMsgBuffer(..., 10) - failed with -1 (true), message was:
transport
```

**Comment by falkTX** [comment]
@nick87720z can you take a look to see if you can reproduce?
thanks

**Comment by nick87720z** [comment]
I tried this: added 4 different plugins, connected them in chain. During setup i had to arrange them. No crash. I tried this both on current develop and that commit. While on that commit, i also tried on external patchbay - to move yoshimi node, also no crashes.

Besides of this thread, there is however failed assertion, when i try to move up/down rack nodes, when some plugin has set custom skin (may be color too), which leads skin/color confs to not move together plugins (i.e., plugins have exchanged not only positions, but skin/color properties too).
